apps/prairielearn/src/lib/grading.ts

```diff
@@ -466,6 +488,7 @@ export async function saveAndGradeSubmission(
     submissionData.user_id,
     submissionData.auth_user_id,
     overrideGradeRateCheck,
+    overrideRealTimeGradingDisabled,
```

Comment:
Nit: these function calls are massive -- can we turn these into all fields on an object?

Reply:
Agreed; I wanted to minimize changes in this PR. Are you comfortable saving this for the future? If not, I'll split this work into a separate PR.

Reply:
Yes, that sounds good.